esp/scm/ws_dali.ecm

```diff
@@ -17,9 +17,32 @@
 
 EspInclude(common);
 
+ESPrequest [nil_remove] SetValueRequest
+{
+    string Path;
+    string Value;
+};
+
+ESPresponse [exceptions_inline, nil_remove] SetValueResponse
+{
+    string Result;
+};
+
+ESPrequest [nil_remove] GetValueRequest
+{
+    string Path;
+};
+
+ESPresponse [exceptions_inline, nil_remove] GetValueResponse
```

Comment:
I wonder since the same semantics (value of prop), if SetValueResponse/GetValueResponse should be commoned up (e.g. to ValueResponse)